### ISSUE TITLE
[FR] Localization support

### ISSUE THREAD

**Issue by MrDini123** [gerbera/gerbera#247]
Hi,

I'd like to see Gerbera and all its components in my mother language. Are there any plans about localization support in the future?

**Comment by v00d00** [comment]
Which areas are you thinking about specifically?

I guess this is mostly a WebUI question, so I will tag @elmodaddyb here.

As for the main daemon, we dont really have that many language specific things (info comes from your files etc), and the things we hard code in "builtin" layout can be replaced in the "js" layout.

**Comment by MrDini123** [comment]
Yes, the webUI is the most important part. But I'd translate the DLNA client directory listing folder names too.

**Comment by elmodaddyb** [comment]
# Awesome idea.  

### Source Code Localization
There are `16` instances of localized text in the Gerbera UI source modules `gerbera*.js`.  Simple status messages like 'Saved' or 'Failed to save'.  These can be externalized to a **locale** file retrieved via the server.

### Template Localization
There are `36` instances of localized text in the Gerbera UI `index.html`.  These can be externalized and retrieved from the server.

**On Load** of the main UI the system can initialize these values via `GERBERA.App.initialize()` using a new `RequestHandler`.

_My native language is English, but also know Russian._

There are many messages from the server, which the UI shows without modification.  Some level of server localization would be required too.

**Comment by elmodaddyb** [comment]
The wikimedia l18n jquery plugin seems popular...but its kinda heavy weight on packages needed.

https://github.com/wikimedia/jquery.i18n

**Comment by v00d00** [comment]
@elmodaddyb Can you make sure whatever we come up with works both the C++ and the JS. Seems like it would be a pain for translators otherwise.

**Comment by elmodaddyb** [comment]
Yea, going to try and find a consolidated way to provide translations along with locale bundles for both sides of the house!

I'll post a few options once done researching.

**Comment by elmodaddyb** [comment]
I was looking at the GNU Translation Project's [`gettext`](https://www.gnu.org/software/gettext/) library.  It seems to be widely available and provides standard POT file which can be pushed through translation engines?

Any thoughts?

**Comment by elmodaddyb** [comment]
Hello All,

I am thinking to pick this feature up as the next set of capability. There is common web handler logic design required for this feature and next phase of the streaming radio playlists (allowing additions from the web UI). 

> Is there still interest in the internationalization support? Feedback welcome.

**Comment by MrDini123** [comment]
Sorry for the huge delay! 😊 

Of course there is. Localization would be awesome.

**Comment by elmodaddyb** [comment]
Well timed.  I recently completed custom icon support (_PR coming soon_), which permits an understanding of custom request handlers.  This pivots nicely to language support.

> This feature will likely be staged like below:

1. Externalize all UI literals to a PO file
2. New `/content/language` handler for retrieving the literals
3. The [`gettext`](https://www.gnu.org/software/gettext/manual/gettext.html) library on the server to load PO files and for server-side language.
